README.md

```diff
@@ -2,14 +2,56 @@
 
 Repository gathering useful libraries and contracts.
 
-## Setup
+## Getting started
+
+Install this repository as a forge dependency:
+
+```bash
+forge install https://github.com/morpho-dao/morpho-utils
+```
+
+Then add a remapping to your `remappings.txt`:
+
+```txt
+@morpho-utils/=lib/morpho-utils/src/
+```
+
+You're good to go! Just import libraries with `@morpho-utils/math/Math.sol` for example.
+
+---
+
+## Development
 
 After cloning this repository, run:
 
 ```bash
 git submodule update --init --recursive
 ```
 
+Configure your VSCode to automatically format a file on save, using `forge fmt`:
+
+- Install [emeraldwalk.runonsave](https://marketplace.visualstudio.com/items?itemName=emeraldwalk.RunOnSave)
+- Update your `settings.json`:
+
+```json
+{
+  "[solidity]": {
+    "editor.formatOnSave": false
+  },
+  "emeraldwalk.runonsave": {
+    "commands": [
+      {
+        "match": ".sol",
+        "isAsync": true,
+        "cmd": "forge fmt ${file}"
+      }
+    ]
+  }
+}
+```
```

Comment:
Not mandatory to have this here, but ok

Reply:
Where do you want to have it?

Reply:
idk, nowhere ^^
people do whatever they want with their setups

Reply:
Ok I think it's more like a tip than something mandatory so I don't it's a pb

Reply:
yes